helm_deploy/values-dev.yaml

```diff
@@ -18,5 +16,5 @@ generic-service:
       cpu: 250m
       memory: 350Mi
 
-env:
-  HMPPS_AUTH_BASE_URL: "https://sign-in-dev.hmpps.service.justice.gov.uk/auth"
\ No newline at end of file
+  env:
```

Comment:
The indentation has changed which may be part of the problem but when running locally, the variable is HMPPS_AUTH.BASE_URL so will HMPPSAUTH_BASEURL not fail ?